docs/source/examples/workspace/topologies/libvirt-new.yml

```diff
@@ -7,7 +7,7 @@ resource_groups:
       - role: libvirt_node
         name: centos71
         uri: qemu:///system
-        count: 1
+        count: 2
```

Comment:
Changing this count fundamentally changes the tests. If there's a good reason to change this, please explain it here. Thx

Reply:
Nope, just another change that wasn't supposed to be pushed